### ISSUE TITLE
weight_norm warning

### ISSUE THREAD

**Issue by Capsar** [paul-krug/pytorch-tcn#1]
Thanks for creating the package & just a heads up,

When using the standard weight_norm with newer versions of PyTorch it gives the following warning:

``` UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")

**Comment by paul-krug** [comment]
I see, thank you for pointing it out. I will fix that in the next release :)

**Comment by paul-krug** [comment]
This was fixed in release 1.1
